bip-0372.mediawiki

```diff
@@ -182,7 +182,7 @@ TBD
     <https://petertodd.org/2016/opentimestamps-announcement>
 [4] Adam Back, Matt Corallo, Luke Dashjr, et al. Enabling Blockchain
     Innovations with Pegged Sidechains (commit5620e43). Appendix A.
-    <https://blockstream.com/sidechains.pdf>;.
+    <https://blockstream.com/sidechains.pdf>.
```

Comment:
While here, perhaps worth updating the non-functional links to mediawiki format so that they work correctly.